editor/src/hooks/isReadOnly.ts

```diff
@@ -0,0 +1,6 @@
+import {useHashParam} from "@metapages/hash-query";
+
+export const useIsReadOnly = () => {
+	const [readOnlyParam] = useHashParam("readonly", undefined);
+	return readOnlyParam !== undefined;
```

Comment:
Instead you can use `useHashParamBoolean` (you may have to update to the latest version of `@metapages/hash-query`, or merge `main` which I think is up-to-date).

Reply:
Using `useHashParamBoolean` always returned `false` for me even when the hash param was there.

I upgraded the library to `0.5.1`, but that version didn't export any `use` hooks - only hash param getters/setters